contentcuration/contentcuration/frontend/channelEdit/components/edit/ActivityDuration.vue

```diff
@@ -153,8 +151,14 @@
           return getShortActivityDurationValidators().map(translateValidator);
         } else if (this.selectedDuration === DurationDropdownMap.LONG_ACTIVITY) {
           return getLongActivityDurationValidators().map(translateValidator);
+        } else if (
+          this.selectedDuration === DurationDropdownMap.EXACT_TIME &&
+          this.selectedCompletion === CompletionDropdownMap.completeDuration
+        ) {
+          return getActivityDurationValidators().map(translateValidator);
+        } else {
```

Comment:
As you're using a return early strategy here, you don't need a final `else` statement.